Source/DeckSegue.swift

```diff
@@ -13,13 +13,15 @@ import UIKit
 /// To use this, set your segue's class to `DeckSegue`, and its `kind` to
 /// `custom`
 public final class DeckSegue: UIStoryboardSegue {
-    
-    var transition: UIViewControllerTransitioningDelegate?
-    
+
+    /// You can customize the segue transition by providing your own DeckTransitioningDelegate in 'prepare(for segue: sender:)'
+    /// i.e. (segue as? DeckSegue)?.transitioningDelegate = DeckTransitioningDelegate(isSwipeToDismissEnabled: false)
+    public var transitioningDelegate: DeckTransitioningDelegate?
```

Comment:
One request here: Could you have the comment wrap at 80 chars? I should look into having that be codified/checked somehow, that's the style used in the rest of the repo.

Reply:
Sure thing. Pushed up updated commit.

I didn't bump the version in case you have other things you'd like to include into it, but if you do want to release this in the new version, the change log would be:

```
## 2.1.1 Release notes (18/9/2018)
----

- Added ability to customize the segue when using with storyboards
```